docs/guides/cloud/introduction.mdx

```diff
@@ -4,153 +4,131 @@ sidebar_position: 10
-securely with
-[Self-managed GitLab](https://docs.gitlab.com/ee/subscriptions/self_managed/)
-instances too.
+results in Cypress Cloud for fast debugging.
 
 <DocsImage
   src="/img/dashboard/github-integration/status-checks-per-spec.jpg"
   alt="Status checks per spec"
 />
 
-### Collaborate and organize
+[Business and Enterprise plans](https://www.cypress.io/pricing/) include secure
+integration with self-managed
+[GitHub](https://docs.github.com/en/enterprise-server@3.5/admin/overview/about-github-enterprise-server)
+and [GitLab](https://docs.gitlab.com/ee/subscriptions/self_managed/) instances
+too.
 
-We also integrate with two of the world's most popular collaboration tools:
-[Slack](https://slack.com/) and [Jira](https://www.atlassian.com/software/jira).
-Deliver test results with valuable additional context, directly into a dedicated
-Slack channel. Or for those on the
-[Team, Business or Enterprise plans](https://www.cypress.io/pricing/),
-[create bidirectionally linked Jira tickets](/guides/cloud/integrations/jira#Creating-a-Jira-issue-for-a-test-case)
-directly from specific test failures.
+#### Team Communication
+
+Get notified on test progress directly in your team's
+[Slack](/guides/cloud/integrations/slack) or
+[Microsoft Teams](/guides/cloud/integrations/teams) channels.
 
 <DocsImage
   src="/img/dashboard/slack-integration/cypress-slack-integration-channel-feed.jpg"
   alt="Cypress notification feed in Slack channel"
   width={560}
 />
 
-Lastly you can use our flexible adminstrative functions to configure Cypress
-Cloud however you want, grouping [projects](/guides/cloud/projects) into
-multiple [organizations](/guides/cloud/organizations) if you have a lot,
-checking [usage](/guides/cloud/organizations#Billing-Usage), and administering
-[users and permissions](/guides/cloud/users#User-roles). We also provide
-[SSO integration](/guides/cloud/organizations#Enterprise-SSO) for teams on our
-[Enterprise plan](https://www.cypress.io/pricing/).
+#### Issue Management
 
-:::info
+Keep the whole team up-to-date with [Jira](/guides/cloud/integrations/jira)
+integration. You can
+[create bidirectionally linked Jira tickets](/guides/cloud/integrations/jira#Creating-a-Jira-issue-for-a-test-case)
+directly from specific test failures.
 
-<strong>Have a question you don't see answered here?</strong>
+### Cypress app integration
 
-[We have answered some common questions about Cypress Cloud in our FAQ](/faq/questions/cloud-faq).
+The [Cypress app](/guides/cloud/runs#Cypress-App) integrates directly with
+Cypress Cloud to provide developers with the latest test results without having
+to switch tools.
 
-:::
+<DocsImage
+  src="/img/guides/core-concepts/cypress-app/debug-page.png"
+  alt="Debug"
+/>
 
-## Example projects
+## Example project
 
-Once you log in to [Cypress Cloud](https://on.cypress.io/cloud) you can view any
-[public project](/guides/cloud/projects#Public-vs-Private).
+The Cypress
```

Comment:
Was removing the callout treatment we use for RWA intentional?

Reply:
If you meant the one that was at the top of the guide, then yes. It felt out of place and better to link down in the example. I took out the tip callout as it was in its own section.

As for the other examples removed, I did so because either the projects don't get many updates or have been deprecated. Most of the commits were from bots and it looked stale.